event/views.py

```diff
@@ -53,4 +50,20 @@ def create_ingroup(request, group_id):
         return render(request, "event/create_ingroup.html", {
             'form': CreateGroupEventForm(),
             'group_id' : group_id
-        }) 
\ No newline at end of file
+        }) 
+    
+def event_profile(request, event_id):
+    event = Event.objects.get(pk=event_id)
+    
+    return render(request, "event/profile.html", {
+        "event" : event,
+        "request_exists" : event.requests.filter(user=request.user.id).exists()
+    })
+
+def group_event_profile(request, group_event_id):
+    groupEvent = GroupEvent.objects.get(pk=group_event_id)
+    
+    return render(request, "event/profile.html", {
+        "event" : groupEvent,
+        "request_exists" : groupEvent.requests.filter(user=request.user.id).exists()
```

Comment:
Does this not give error since there are no GroupEvent.requests? I think we will also need to update the model